### ISSUE TITLE
Implement Twitter Login

### ISSUE THREAD

**Issue by Nick-Case** [csblogs/csblogs#109]
Some bloggers may not use wordpress, possibly not GitHub (especially at Hull), or stack exchange. Catering for everyone is impossible, but a large amount of people use Twitter.

**Comment by AllFallD0wn** [comment]
Started in f7f81886a5c5388e750c57a7a4074ac9b169ca8a

**Comment by Nick-Case** [comment]
Started in f7f81886a5c5388e750c57a7a4074ac9b169ca8a

**Comment by Lewis-T** [comment]
facebook & goolge? its all Oauth2 right?

**Comment by Nick-Case** [comment]
Yup. Twitter is basically done, just waiting on logos for front end. Facebook &  Google can follow next
